packages/atomic/src/components/insight/atomic-insight-result/atomic-insight-result.pcss

```diff
@@ -5,3 +5,21 @@ atomic-result-section-children {
   border-left: 1px solid var(--atomic-neutral);
   padding-left: 1rem;
 }
+
+:host(.resultHovered) {
+  @apply relative;
+
+  .with-sections {
+    background-color: var(--atomic-neutral-lighter);
+  }
+}
+
+:host {
+  .with-sections {
+    @apply p-6;
+  }
+}
+
+:host::before {
+  @apply my-0 mx-6 !important;
```

Comment:
:finnadie: `!important` is a pain to deal with and often a code smell.
Can you try to increase the specificity of your CSS selector instead?